engine/message/db.go

```diff
@@ -575,6 +594,49 @@ func (db *DB) _SendMessages(ctx context.Context, send SendFunc, status StatusFun
 		return errors.Wrap(err, "clear disabled status updates")
 	}
 
+	// if twilio is disable, create an entry to notify the user
+	cfg := config.FromContext(ctx)
+	if !cfg.Twilio.Enable {
+		rows, err := tx.Stmt(db.twilioDisabled).QueryContext(execCtx)
+		if err != nil {
+			return errors.Wrap(err, "check for failed message")
+		}
+		defer rows.Close()
+
+		type record struct {
```

Comment:
use `msgMeta` below by raising it above, along with the `var msgs msgMeta` declaration

Both of these can just append to the same slice that way and we can use the single `for _, m := range msgs {` below